src/www/ui/api/documentation/openapi.yaml

```
@@ -1728,6 +1728,56 @@ paths:
         default:
           $ref: '#/components/responses/defaultResponse'
 
+/file/upload/{id}/item/{ItemId}/copyrights/inactive:
```

Please be consistent with other endpoints
```suggestion
  /uploads/{id}/items/{itemId}/copyrights:
```

---

Would suggest to use a query parameter like `filter` with enum `active, inactive` rather than creating separate endpoints to get active and inactive copyrights.

---

Yes sir, I will be implementing this and updating one of the PRs for this operation

---

> Would suggest to use a query parameter like `filter` with enum `active, inactive` rather than creating separate endpoints to get active and inactive copyrights.

This PR's feature is implemented in #2475 using query parameters. 

@GMishx please do have a look when possible.

---

So can I close this in favor of #2475 ?

---

Sure sir!